subsys/mgmt/Kconfig

```diff
@@ -57,18 +56,60 @@ config MCUMGR_SMP_UART_MTU
 
 endif
 
+config MCUMGR_SMP_UDP
+	bool "UDP mcumgr SMP transport"
+	select MCUMGR
+	select NETWORKING
+	select NET_IPV4
+	select NET_UDP
+	select NET_SOCKETS
+	help
+	  Enables handling of SMP commands received over UDP.
+	  Will start a thread for listening on the configured UDP port.
+
+if MCUMGR_SMP_UDP
+config MCUMGR_SMP_UDP_PORT
+	int "UDP SMP port"
+	default 1337
+	help
+	  UDP port that SMP server will listen for SMP commands on.
+
+config MCUMGR_SMP_UDP_STACK_SIZE
+	int "UDP SMP stack size"
+	default 512
+	help
+	  Stack size of the SMP UDP listening thread
+
+config MCUMGR_SMP_UDP_THREAD_PRIO
+	int "UDP SMP thread priority"
+	default 0
+	help
+	  Scheduling priority of the SMP UDP listening thread.
+
+config MCUMGR_SMP_UDP_MTU
+	int "UDP SMP MTU"
```

Comment:
This does not seem to be a network MTU but just some max buffer size when receiving data. Please adjust the name and remove the MTU as it is quite confusing to read.

Reply:
The term MTU is generally used in the mgmt SMP subsys, so the naming I chose is based on existing values like `MCUMGR_SMP_SHELL_MTU`. Do you still want me to change the naming?

Reply:
It the term is used generally in the SMP, then that is fine.